src/__mocks__/axiosResponses.ts

```diff
@@ -0,0 +1,34 @@
+import { AxiosError, AxiosResponse, AxiosRequestConfig } from 'axios'
+
+export let userData:object = {
+  username: 'test',
+  firstname: 'testfn',
+  lastname: 'testln',
+  email: 'testemail',
+  password: 'testpass',
+}
+
+export const successResponseMessageRegister: AxiosResponse = {
```

Comment:
Perhaps we can scope the folders so that we can shorten the variable name like
`src/__mocks__/Register/index.ts` and this can be `successResponse/errorResponse`, what do you think?